server/src/application/application.entity.ts

```diff
@@ -0,0 +1,53 @@
+import { User } from '@/user/user.entity';
+import { Entity, PrimaryGeneratedColumn, Column, JoinColumn, OneToOne, ManyToOne } from 'typeorm';
+import { CustomBaseEntity } from '../common/custom-base.entity';
+import {
+  ContactInfo,
+  GeneralProjectInfo,
+  FacilityInfo,
+  FundingEligibility,
+  ClimateConsiderationsAirside,
+  ClimateConsiderationsEnvironmental,
+  ProjectBenefits,
+  ProjectFunding,
+  SupportAndAuthorization,
+} from './application.interfaces';
+
+@Entity()
+export class Application extends CustomBaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'varchar', length: 100, nullable: false, unique: true })
+  externalId: string;
```

Comment:
This will be in the User table

Reply:
Woops, thank you! Should be addressed now.